lib/barbeque/slack_notifier.rb

```diff
@@ -0,0 +1,71 @@
+require 'barbeque/slack_client'
+
+module Barbeque
+  class SlackNotifier
```

Comment:
[nits] As it doesn't have ivar like `SlackClient` instance or some state for now, I would define this as module and define all methods as class methods to make it clear that it doesn't have such things.